File: clients/Java-GraalVM-native/src/main/java/model/RepairAction.java
package model;

import util.StreamUtil;

public class RepairAction {
    private int target;
    public int getTarget() { return target; }
    public void setTarget(int target) { this.target = target; }
    public RepairAction() {}
    public RepairAction(int target) {
        this.target = target;
    }
    public static RepairAction readFrom(java.io.InputStream stream) throws java.io.IOException {
        RepairAction result = new RepairAction();
        result.target = StreamUtil.readInt(stream);
        return result;
    }
    public void writeTo(java.io.OutputStream stream) throws java.io.IOException {
        StreamUtil.writeInt(stream, target);
    }
}

File: clients/Kotlin-GraalVM-native/src/main/kotlin/model/RepairAction.kt
package model

import util.StreamUtil

class RepairAction {
    var target: Int = 0
    constructor() {}
    constructor(target: Int) {
        this.target = target
    }
    companion object {
        @Throws(java.io.IOException::class)
        fun readFrom(stream: java.io.InputStream): RepairAction {
            val result = RepairAction()
            result.target = StreamUtil.readInt(stream)
            return result
        }
    }
    @Throws(java.io.IOException::class)
    fun writeTo(stream: java.io.OutputStream) {
        StreamUtil.writeInt(stream, target)
    }
}
